### DISCUSSION TITLE
wds on newer Fritzbox

### DISCUSSION THREAD

**Discussion by jkobber** [Freetz-NG/freetz-ng#605] [Allgemein]
Ich habe eine Fritz!Box 7490 welche nicht mehr WDS hat und will sie in ein WDS mit anderen custom <a href="https://advancedtomato.com/">Tomato</a>  Routern verbinden. Geht das mit freetz?

**Comment by fda77** [comment]
Freetz ändert nichts am wlan. Du kannst die Fritzbox aber als ip-client ("internet mitbenutzen") konfigurieren und dann als Access-Point nutzen
